DQMServices/Demo/plugins/DemoNormalDQMEDAnalyzer.cc

```diff
@@ -70,7 +70,7 @@ void DemoNormalDQMEDAnalyzer::bookHistograms(DQMStore::IBooker& ibook,
                                              edm::EventSetup const& iSetup) {
   ibook.setCurrentFolder(folder_);
 
-  example_ = ibook.book1D("EXAMPLE", "Example 1D", 20, 0., 10.);
+  example_ = ibook.book1D("EXAMPLE", "Example 1D", 20, 0., 10., [](TH1*) { std::cout << "booked!\n"; });
```

Comment:
How about using MessageLogger?

Reply:
Oh indeed, that was a very quick test if the new callbacks work...